docs/source/api/cdns_routing.rst

```diff
@@ -33,12 +33,16 @@ No parameters available
 
 Response Structure
 ------------------
-:cz:          Used Coverage Zone geographic IP mapping
-:dsr:         Overflow traffic sent to secondary CDN
-:err:         Error localizing client IP
-:geo:         Used 3rd party geographic IP mapping
-:miss:        No location available for client IP
-:staticRoute: Used pre-configured DNS entries
+:cz:                The percent of requests to the Traffic Router for this :term:`Delivery Service` that were satisfied by a :term:`Coverage Zone File`
+:deepCz:            The percent of requests to the Traffic Router for this :term:`Delivery Service` that were satisfied by a Deep :term:`Coverage Zone File`
+:dsr:               The percent of requests to the Traffic Router for this :term:`Delivery Service` that were satisfied by sending the client to an overflow :term:`Delivery Service`
+:err:               The percent of requests to the Traffic Router for this :term:`Delivery Service` that resulted in an error
+:fed:               The percent of requests to the Traffic Router for this :term:`Delivery Service` that were satisfied by sending the client to a federated CDN
+:geo:               The percent of requests to the Traffic Router for this :term:`Delivery Service` that were satisfied using 3rd party geographic IP mapping
```

Comment:
you could put the "rd" into superscript if you'd like. It's just a contraction, so there's nothing grammatically wrong with what you have here.

```rst
... 3\ :sup:`rd`...
```
renders like: 3<sup>rd</sup>

Reply:
i suppose i'll just leave it